src/Authentication/Authentication/Models/GraphEnvironment.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph.PowerShell.Authentication
```

Comment:
haven't noticed before but the namespaces don't follow the folder structure.  I expected this to be in .models.